docs/pages/machine-id/workload-identity/getting-started.mdx

```diff
@@ -0,0 +1,171 @@
+
+## Step 1/4. Configure RBAC
+
+First, Teleport must be configured to allow the Bot to issue SPIFFE SVIDs.
+This is done by configuring a role to allow the issuance of SPIFFE SVIDs and
+then granting this role to the Bot.
+
+Before proceeding, you'll want to determine the SPIFFE ID path that your
+workload will use. In our example, we'll use `/svc/foo`. We provide more
+guidance on choosing a SPIFFE ID structure in the
+[Best Practices](./best-practices.mdx) guide.
+
+Create a new file named `spiffe-issuer-role.yaml`:
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: svid-issuer
+spec:
+  allow:
+    spiffe:
+    - path: "/svc/foo"
+```
+
+Replace:
+
+- `spiffe-issuer` with a name that describes your use-case.
+- `/svc/foo` with the SPIFFE ID path you have decided on issuing.
+
+Use `tctl create -f ./spiffe-issuer-role.yaml` to create the role.
+
+Now, use `tctl bots update` to add the role to the Bot. Replace `example-bot`
+with the name of the Bot you created in the deployment guide and `spiffe-issuer`
+with the name of the role you just created:
+
+```bash
+$ tctl bots update example-bot --add-roles spiffe-issuer
+```
+
+### Step 2/4. Configure `spiffe-workload-api` service in `tbot`
+
+To set up a SPIFFE Workload API endpoint with `tbot`, we configure an instance
+of the `spiffe-workload-api` service.
+
+First, determine where you wish this socket to be created. In our example,
+we'll use `/opt/machine-id/workload.sock`. You may wish to choose a directory
+that is only accessible by the processes that will need to connect to the
+Workload API.
+
+Modify your `tbot` configuration file to include the `spiffe-workload-api`
+service:
+
+```yaml
+services:
+- type: spiffe-workload-api
+  listen: unix:///opt/machine-id/workload.sock
+  svids:
+  - path: /svc/foo
+    hint: my-hint
+```
+
+Replace:
+
+- `/opt/machine-id/workload.sock` with the path to the socket you wish to create.
+- `/svc/foo` with the SPIFFE ID path you have decided on issuing.
+- `my-hint` with a hint that will be included with the SVID. This can help
+  workloads identify which SVID they should select if multiple are presented.
+  This field can be omitted if not required.
+
+Start or restart your `tbot` instance to apply the new configuration
+
+#### Configuring DNS and IP SANs
+
+In some cases, you may wish to configure DNS and IP SANs which should be
+included in the SVIDs issued by the Workload API. This is useful in cases
+where the client may not be SPIFFE aware and will check the DNS SAN rather than
+the SPIFFE URI during the TLS handshake.
+
+These can be configured in the `spiffe-workload-api` service using the `sans`
+configuration block:
+
+```yaml
+services:
+- type: spiffe-workload-api
+  listen: unix:///opt/machine-id/workload.sock
+  svids:
+  - path: /svc/foo
+    hint: my-hint
+    sans:
+      dns:
+      - example.com
+      ip:
+      - 10.0.0.1
+```
+
+### Step 3/4. Testing the Workload API with `tbot spiffe-inspect`
```

Comment:
Step 3 and 4 would make more sense as H2s, since the H2 that precedes them is Step 1.

Reply:
2285838b13